components/_global-navigation.scss

```diff
@@ -193,12 +210,24 @@ $notification-counter-size: 18px;
 		flex: 0 1 auto;
 	}
 
+	&__search {
+		flex: 0 1 auto;
+		transition: flex $search-transition-duration;
+	}
+
 	&__search-input-wrapper {
-		border: 0 solid $color-slate-gray;
+		border: {
+			bottom: 1px solid transparent;
+			left: 0;
+			right: 0;
+			top: 0;
+		};
```

Comment:
I like Damian's suggestion, this notation is more clear. Another alternative is:

```
border: solid transparent;
border-width: 0 0 1px 0;
```

which I like most ;)